src/tools/perf/lib/libperf_int.h

```diff
@@ -221,6 +222,12 @@ static UCS_F_ALWAYS_INLINE void ucx_perf_update(ucx_perf_context_t *perf,
     }
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_perf_is_daemon_mode(const ucx_perf_params_t *params)
+{
+    return (0 != params->ucp.dmn_local_addr.ss_family);
```

Comment:
1. use AF_UNSPEC instead of 0
2. maybe there could be a better way to check if it's a daemon? like some flag?